openedx_caliper_tracking/script.py

```diff
@@ -14,7 +14,7 @@
 TESTS_LOG_FILE = 'caliper_tests.log'
 
 
-logger = get_caliper_logger('caliper_tests', TESTS_LOG_FILE)
+logger = get_test_logger('caliper_tests', TESTS_LOG_FILE)
```

Comment:
What is the purpose of the test logger?

Reply:
The caliper test script logs using this logger

Reply:
Oka.